buildenv/jenkins/jobs/infrastructure/Update-Reference-Repos.groovy

```diff
@@ -141,7 +141,7 @@ timeout(time: 6, unit: 'HOURS') {
 
                         if (jenkins.model.Jenkins.instance.getLabel(SETUP_LABEL).getNodes().contains(aNode)) {
                             // add OpenJ9 repo
-                            repos.addAll(EXTENSIONS_REPOS)
+                            repos.add([name: "openj9", url: VARIABLES.openj9.get('default').get('repoUrl')])
```

Comment:
This could introduce duplicates, thus you could check if `repos `contains openj9 repo or add `repos.unique()` after this if statement (at L147)

Reply:
Good catch, added.